extension/content.js

```diff
@@ -117,9 +133,14 @@ function moveVotes() {
 
 		const isUp = commentIsUseless('upvote', el);
 		const isDown = commentIsUseless('downvote', el);
-		const commenter = $(el).closest('.js-comment-container').find('.author').get(0).innerHTML;
 
 		if (isUp || isDown) {
+			// grab avatar and wrapping a tag for commenter
+			const commenterHTML = $(el).closest('.js-comment-container').find('a').get(0);
+			const tmp = document.createElement('div');
+			$(tmp).append(commenterHTML);
+			const commenter = tmp.innerHTML;
```

Comment:
It'd actually be more like the following, but even then it doesn't work. This gets me back to a DOM node and I don't want that, I just want a string to shove into the DOM.

```js
$(commenterHTML).wrap('<div>').dom[0];
```